Sources/CocoaExtensions/NSControl.swift

```diff
@@ -0,0 +1,8 @@
+import Cocoa
```

Comment:
Perhaps we could have a module like "PlatformExtensions" or "CrossPlatformExtensions" where we would have things analogous to what we currently do in SKExtensions, e.g.:
```
#if canImport(AppKit)
    public typealias SKWindow = NSWindow
#elseif canImport(UIKit)
    public typealias SKWindow = UIWindow
#endif
```
Alternatively, we could just continue use SKExtensions so that it also plays the role of something like CrossPlatformExtensions?

Reply:
There is no equivalent that I want to add for anything other than Cocoa. UIKit and SwiftUI already have nice APIs.

Reply:
Now that you made `CrossPlatformExtensions`, I'm wondering if our Cocoa extensions can be included there too so that we only have one module for providing cross-platform api support?